src/control/cmd/daos/mock_daos.go

```diff
@@ -0,0 +1,67 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
I'm not sure about this approach. I believe @mjmac still has a work in progress for making the underlying libdaos something mockable. I'd avoid adding something like this for now.